state-sync/src/coordinator.rs

```diff
@@ -866,12 +863,7 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
             .chunk_response(chunk_response.clone())
             .peer(&peer);
         let msg = StateSyncMessage::GetChunkResponse(Box::new(chunk_response));
-
-        let network_sender = self
-            .network_senders
-            .get_mut(&peer.network_id())
-            .expect("missing network sender");
-        let send_result = network_sender.send_to(peer.peer_id(), msg);
+        let send_result = self.request_manager.send_chunk_response(&peer, msg);
```

Comment:
Lol, another thing where the responsibilities just didn't line up, thanks for catching this.